internal/orchestra/testlists/tor/tor.go

```diff
@@ -0,0 +1,42 @@
+// Package tor contains code to fetch targets for the tor experiment.
+package tor
+
+import (
+	"context"
+
+	"github.com/ooni/probe-engine/model"
+)
+
+// Config contains settings.
+type Config struct{}
+
+var staticTestingTargets = []model.TorTarget{
+	// TODO(bassosimone): this is a public working bridge we have found
+	// with @hellais. We should ask @phw whether there is some obfs4 bridge
+	// dedicated to integration testing that we should use instead.
```

Comment:
@nullhypothesis do you have tips for what better bridge we could use in here?

Reply:
The most reliable bridges are our default bridges because we monitor them and poke their operators as soon as one goes offline:
https://trac.torproject.org/projects/tor/wiki/doc/TorBrowser/DefaultBridges
You may want to pick David Goulet's bridge `CDF2E852BF539B82BD10E27E9115A31734E378C2`. It has been online for a long time, it's reliable, and it's easy to get in touch with David.

Reply:
done!